code/utils/parse_demographics.py

```diff
@@ -0,0 +1,121 @@
+import csv
+
+
+class Subject:
+    def __init__(self, demographics):
+        self.id = int(demographics["id"])
+        self.gender = demographics["gender"]
+        self.age_range = demographics["age"]
+        if demographics["forrest_seen_count"]:
+            self.forrest_seen_count = int(demographics["forrest_seen_count"])
+        # Accounting for case of missing data
+        else:
+            self.forrest_seen_count = -1
+
+
+def parse_csv(fname):
+    """
+	Parses the given demographics.csv file and creates instances for each
+    subject. Returns subject instances in a list.
+
+	Parameters
+	----------
+	fname : string
+
+	Returns 
+	-------
+	subjects : array
+	"""
+    subjects = []
+    with open(fname) as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            subjects.append(Subject(row))
+        return subjects
```

Comment:
use python generator expressions:

with open(fname) as csvfile:
    reader = csv.DictReader(csvfile)
    return [Subject(row) for row in reader]